src/feature_collection.rs

```diff
@@ -47,6 +48,21 @@ use crate::{util, Bbox, Feature};
 /// );
 /// # }
 /// ```
+///
+/// Collect from an iterator:
+///
+/// ```rust
+/// # extern crate geojson;
+/// # fn main() {
```

Comment:
nbd: this `main` declaration can go away, right?

https://doc.rust-lang.org/rustdoc/documentation-tests.html#pre-processing-examples

(thanks for writing an example doc though!)

Reply:
I just copied the previous example.  I guess it's for the `extern crate` line above `fn main()`.

Reply:
Seems to work without the `/// # ` lines, so have removed them from both examples.